packages/go/slicesext/slicesext.go

```diff
@@ -125,12 +118,20 @@ func Init[T any](list []T) []T {
 	return list[:len(list)-1]
 }
 
-// Reverse reverses the order of a slice by doing an in place reverse
-// This will reorder the provided slice in place, and uses zero allocations
-func Reverse[T any](list []T) []T {
-	for low, high := 0, len(list)-1; low < high; low, high = low+1, high-1 {
-		list[low], list[high] = list[high], list[low]
+// Concat returns a new slice concatenating the passed in slices.
+// This was ripped from go1.22 source and should be replaced with the stdlib
```

Comment:
Nice lol

Reply:
Do we need to do any license stuff here if we're admitting to taking this directly from go source?

Reply:
Hmm, that's a great question that I probably should have thought through. A link to the original is likely due at a minimum

Reply:
Fixed up